src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -59,8 +59,8 @@
 #include "oops/method.inline.hpp"
 #include "oops/objArrayKlass.hpp"
 #include "oops/objArrayOop.inline.hpp"
-#include "oops/oop.inline.hpp"
 #include "oops/oop.hpp"
+#include "oops/oop.inline.hpp"
```

Comment:
Take this opportunity to remove the include of `oop.hpp`, since we are including `oop.inline.hpp` anyway?